### PR TITLE
Issue #4769: Control coverage of JavadocParser and JavadocLexer

### PR DESCRIPTION
**Pull request by Vladlis**
#4769
JavadocParserListener and JavadocParserVisitor are excluded from generation as they are not used at all.

### CODE REVIEW

**Comment by rnveach** [comment]
Coverage failed.
> Classes which are missed in Cobertura coverage report:
com/puppycrawl/tools/checkstyle/gui/BaseCellEditor
com/puppycrawl/tools/checkstyle/gui/CodeSelector

@romani Does the shell script need to be updated? https://github.com/checkstyle/checkstyle/blob/master/.ci/travis/travis.sh#L184

**Comment by romani** [comment]
>Does the shell script need to be updated

Might be, but not for gui part

**Comment by romani** [comment]
comment over exclude is required to explain why we keep it without consideration.

File: pom.xml [hunk 1]
```diff
@@ -1524,11 +1525,21 @@
...
+                    <pattern>com.puppycrawl.tools.checkstyle.grammars.javadoc.JavadocLexer</pattern>
+                    <branchRate>41</branchRate>
+                    <lineRate>55</lineRate>
+                  </regex>
                 </regexes>
               </check>
               <instrumentation>
                 <excludes>
-                  <exclude>com/puppycrawl/tools/checkstyle/grammars/javadoc/*.class</exclude>
+                  <exclude>com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser$*.class</exclude>
```
Comment (romani) [inline pom.xml]: comment over exclude is required to explain why we keep it without consideration.

Comment (Vladlis) [inline pom.xml]: Done

**Pull request merged**